packages/create-sitecore-jss/src/templates/nextjs-personalize/src/components/CdpIntegrationScript.tsx

```diff
@@ -60,15 +58,17 @@ const CdpIntegrationScript = ({
           __html: `
               var _boxeverq = _boxeverq || [];
 
+              var currentHostname = window.location.host.replace(/^www\./,'');
```

Comment:
I checked and by default `window.location.host` contains only hostname, `www` shouldn't be there as I can see, does it make sense to replace it?